docs/source/example_coils.rst

```diff
@@ -302,3 +346,263 @@ for coil ``j`` can be obtained from ``coils[j].curve.x``, where the
 meaning of each array element can be seen from
 ``coils[j].curve.dof_names``.  The position vector for coil ``j`` in
 Cartesian coordinates can be obtained from ``coils[j].curve.gamma()``.
+
+Stochastic Optimization
+---------------------------
+
+In this example we solve a stochastic version of
+the :ref:`first example here <_simplest_stage2>`. As before,
+the goal is to find coils that generate a specific target
+normal field on a given surface. As we are still considering a vacuum
+field the target is just zero.
+The target equilibrium is the precise QA configuration of arXiv:2108.03711.
```

Comment:
Could you go through the file and replace all the arxiv references with the doi of the published paper?

Reply:
Done